docs/articles/documentation/using-testcafe/configuration-file.md

```diff
@@ -0,0 +1,715 @@
+---
+layout: docs
+title: Configuration File
+permalink: /documentation/using-testcafe/configuration-file.html
+---
+# Configuration File
+
+TestCafe uses the `.testcaferc.json` configuration file to store its settings.
+
+> Important! Settings you specify when you run TestCafe from the [command line](command-line-interface.md) and [programming interfaces](programming-interface/README.md) override settings from `.testcaferc.json`. TestCafe prints a message in the console when this happens.
```

Comment:
`TestCafe prints a message in the console when this happens.` I think it could be rephrased, we need say we print the information about the overridden properties.